doc/force.com.txt

```diff
@@ -461,10 +446,226 @@ Params:
+See also |'g:apex_conflict_check'|.
+
+                                                                     *ApexQuery*
+
+Execute block of code as SOQL query.
+Block can be either whole current buffer or visual selection.
+Supports 2 (optional) parameters:
+- API: Partner or Tooling
+- project-name
+Example >
+    :ApexQuery Tooling MyOrg
+<
+                                                               *ApexQueryRepeat*
+
+A shortcut to repeat last :|ApexQuery| command.
+Useful when you want to re-run visual selection which is not currently selected,
+or you are not in the buffer where last :|ApexQuery| was run.
+Supports 2 (optional) parameters, see :|ApexQuery|
+
+                                                               *ApexRefreshFile*
+
+Refresh current (single) file and replace it with its version from SFDC.
+
+Note: :|ApexRefreshFile| uses "diffWithRemote" in the background and only 
+updates selected local file but does not touch local timestamp cache, hence 
+during saving / deploying file to SFDC file still will be considered as 
+outdated.
+
+                                                            *ApexRefreshProject*
+
+Refresh current project and download new/modified files from SFDC replacing
+all local files.
+
+                                                              *ApexRemoveStaged*
+
+Delete staged files from current or specified Org. See |ApexStageAdd|.
+Example 1: >
+	:ApexRemoveStaged
+< will remove all files listed in :ApexStage from the current Org
+Example 2: >
+	:ApexRemoveStaged My Other Org
+< will remove staged files from "My Other Org"
+
+See also :|ApexDeployDestructive|
+
+                                                                  *ApexRetrieve*
+
+Invoke metadata types list "menu" and load data of specified types.  
+Usually :|ApexRefreshProject| will retrieve all types defined in package.xml.
+:|ApexRetrieve| can be used to load types which are not part of current
+package.xml Use this command when package.xml does not yet contain necessary
+metadata type(s). If retrieved type is not part of the package.xml system will
+offer to update package.xml accordingly. 
+*WARNING* :|ApexRetrieve| command is Experimental. You may want to backup your
+project before running :|ApexRetrieve|.
+
+There are 2 potential scenarios which may involve :|ApexReteieve|
+1. Load all members of given metadata type
+   e.g. to load all Custom Objects you will do something like this >
+	:ApexRetrieve
+< then select CustomObjects line and then run >
+	:Retrieve
+<
+2. Load selected members of given metadata type
+   e.g. to load only Account metadata you will do something like this >
+	:ApexRetrieve
+< then select CustomObjects line and then run >
+	:Expand
+< then select Account and run >
+	:Retrieve
+<
+                                                                      *ApexSave*
+
+Similar to :|ApexDeploy| but uses Tooling API instead of Metadata API.
+Takes modified files and tries to save them to SFDC.
```

Comment:
Would be nice to have "modified" defined here. File timestamp compared to SFDC timestamp? Does plugin persist a last-uploaded time for that file somewhere? Is a hash of that file calculated and compared to the hash of the last time it was uploaded?